rtran-maven/src/main/scala/com/ebay/rtran/maven/util/MavenUtil.scala

```diff
@@ -96,6 +96,12 @@ object MavenUtil {
     val dependencyFilter = new ExclusionsDependencyFilter(
       dependency.getExclusions.map(e => s"${e.getGroupId}:${e.getArtifactId}")
     )
+
+    def recoverFromResolutionException: PartialFunction[Throwable, util.List[Artifact]] = {
```

Comment:
Here is the detail of `recover` from `Try`, I don't think we need to declare `other` for avoiding match exception.
```scala
  def recover[U >: T](rescueException: PartialFunction[Throwable, U]): Try[U] =
    try {
      if (rescueException isDefinedAt exception) {
        Try(rescueException(exception))
      } else this
    } catch {
      case NonFatal(e) => Failure(e)
    }
```